python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -571,7 +637,7 @@ def convert_pooling(node, **kwargs):
 
     kernel = eval(attrs["kernel"])
     pool_type = attrs["pool_type"]
```

Comment:
Can you also add default value for pool_type?

Reply:
Done